src/crypto/__tests__/crypto-alpha-38-test.ts

```diff
@@ -0,0 +1,168 @@
+import * as c from '../crypto-alpha-38'
+import {
+    makeTypedMessage,
+    TypedMessageComplex,
+    TypedMessageUnknown,
+} from '../../extension/background-script/CryptoServices/utils'
+import { derive_AES_GCM_256_Key_From_PBKDF2, import_PBKDF2_Key } from '../../utils/crypto.subtle'
+import { encodeText, encodeArrayBuffer, decodeText } from '../../utils/type-transform/String-ArrayBuffer'
+import { CryptoKeyToJsonWebKey } from '../../utils/type-transform/CryptoKey-JsonWebKey'
+import { recover_ECDH_256k1_KeyPair_ByMnemonicWord } from '../../utils/mnemonic-code'
+
+function helper(x: any) {
```

Comment:
Better change the function name - helper doesn't make any sense.

Reply:
it's a local function, not exported